scripts/apps/authoring/widgets/widgets.ts

```diff
@@ -140,6 +140,8 @@ export const widgetReactIntegration: IWidgetIntegration = {
     disableWidgetPinning: false,
 };
 
+export let closedThroughAction = {closed: true};
```

Comment:
Because of the nature (specific cases where this variable has to be set)  of the fix, I'm not so sure now this can be improved, while using this approach with the variable. And currently I can't think of another way besides using a variable, and handling logic like this in the authoring itself.